src/services/resource-service.ts

```diff
@@ -97,13 +97,25 @@ export const ResourceService = {
   },
   getQuiz: async (id?: string): Promise<AxiosResponse<Quiz>> =>
     await axiosClient.get(createUrlPath(URLs.quizzes.get, id)),
-  addQuiz: async (data?: CreateQuizParams): Promise<AxiosResponse> =>
-    await axiosClient.post(URLs.quizzes.add, data),
-  editQuiz: async (params?: UpdateQuizParams) =>
-    await axiosClient.patch(
-      createUrlPath(URLs.quizzes.patch, params?.id),
-      params
-    ),
+  addQuiz: async (data: CreateQuizParams) => {
```

Comment:
You can remove `async` here since there is no `await` inside
```suggestion
  addQuiz: (data: CreateQuizParams) => {
```